docs/adr/adr_authorization-specification.md

```diff
@@ -0,0 +1,177 @@
+# ADR: Authorization Specification
+
+[Trello card](https://trello.com/c/kypUAjlw/747-20-create-agile-design-documents-artifacts-about-authorization-implementation)
+
+Decision deadline: irrelevant since implementation pending
+
+Author: [Philipp](https://github.com/orgs/boxwise/people/pylipp)
+
+## Status
+
+Implementation on-going.
+
+## Context
+
+The `boxtribute` application is used by various organisations to manage distribution of aid goods. Some data stored contains personal, confidential information and must not be exposed to unintended parties. Implementation of authorization measures is required to define and enforce data excess depending on the current application user.
+
+The current document serves as a summary of decisions about user authorization in the boxtribute application.
+
+## Decision drives
+
+1. Security: exposure of confidential data must be prohibited
+1. Clarity: make the governing structures easy to understand for stakeholders (users, product management, developers)
+1. Single source of trust: data for controlling authorization must not be distributed
+1. Simplicity during development: enforcing authorization should be straightforward for developers while guaranteeing security
+1. Maintainability: management of roles and permissions should have little overhead and be extensible
+1. Integration in both dropapp and boxtribute 2.0
+
+## Authorization in boxtribute
+
+### Fundamental concepts
+
+#### Organisation and base
+
+The `boxtribute` partner organisations operate in one or more sites each, called `bases`. Any registered user belongs to exactly one organisation, and one or more bases subordinated to this organisation. Hence they must not be granted access to any resource outside of the organisation or bases they're assigned to (there are exceptions to this rule depending on the context, e.g. for box transfers).
+
+#### Usergroups
+
+A usergroup reflects the user's responsibilities in the partner organisations. Currently, these usergroups exist:
+
+- administrator (head of operations)
+- coordinator
+- warehouse volunteer
+- free shop volunteer
+- library volunteer
+- label creator
+- God user
```

Comment:
I suggest

> God user (note: this is an application administration role and does not belong to a specific partner organisation)

Because the prior sentence defining usergroups makes it ambiguous for the user.